wordpress-mobile/WordPress-Aztec-iOS
# Implements block-level node finding, and node splitting.

This PR implements several logical instruments we'll need to update the DOM.

- Block-level node finding basically means we can find all lowest (hierarchy-wise) block-level nodes intercepting a range.
- Splitting nodes and their children using a range.

**How to test:**
- Since this PR is mostly non-hooked logical methods, there's not much that can be tested from the GUI yet.
- Run the unit tests and make sure they succeed.
